### PR TITLE
Update polymorphism_1.cs

### PR DESCRIPTION
**Pull request by christos-P**
# Title

Minor changes to make code more readable.

## Summary

Minor changes to make code more readable.

## Details

I propose we change the initialization of shapes using a Collection Initializer, instead of creating a List first and then Add one item after another. Also I propose we change the variable name in foreach from s to shape, since this I think it more meaningful. Last but not least I think using var instead of the exact types System.Collections.Generic.List<Shape> and Shape makes code more readable.

### CODE REVIEW

File: docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs [hunk 1]
```diff
@@ -49,16 +49,18 @@ static void Main(string[] args)
             // can all be used whereever a Shape is expected. No cast is
             // required because an implicit conversion exists from a derived 
             // class to its base class.
-            System.Collections.Generic.List<Shape> shapes = new System.Collections.Generic.List<Shape>();
-            shapes.Add(new Rectangle());
-            shapes.Add(new Triangle());
-            shapes.Add(new Circle());
+            var shapes = new System.Collections.Generic.List<Shape>
```
Comment (svick) [inline docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs]: Would it make sense to also simplify `System.Collections.Generic.List<Shape>` to just `List<Shape>`?

Comment (christos-P) [reply docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs]: I totally agree with you ! Furthermore, I would suggest we include the using statement,
using System.Collections.Generic, at the top of our source file, in order when someone copy and paste it in an environment, in order to test it not to forget it.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for modernizing this code example, as well as for your contribution to the dotnet/docs repo, @christos-P. Just two changes -- add a `using` statement, and remove the namespace names in the List\<Shape> instantiations.

File: docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs (hunk 1 above)
Comment (svick) [reply docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs]: In that case, I think that `using System;` (required for `Console`) should be added as well.

**Comment by christos-P** [comment]
@rpetrusha You are very welcome. I just dided the corresponding changes and I have commited them.

**Review by svick — APPROVED** [review]

**Review by rpetrusha — APPROVED** [review]
Thanks for making the additional changes, @christos-P. We'll merge your PR now. The updated code should appear on docs.microsoft.com within a day or two.

**Comment by christos-P** [comment]
@rpetrusha You are welcome !

**Pull request merged**